plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareStorageProcessor.java

```diff
@@ -915,7 +915,7 @@ private String cloneVMforVvols(VmwareContext context, VmwareHypervisorHost hyper
         if (!_fullCloneFlag) {
             createVMLinkedClone(vmTemplate, dcMo, vmName, morDatastore, morPool);
```

Comment:
"disk provisioning type" not considered in case of linked clone ?

Reply:
I have added checks